.github/PULL_REQUEST_TEMPLATE.md

```
@@ -0,0 +1,80 @@
+<!--
+Thank you for the PR. Please fill every section — write "N/A" if a section does
+not apply. Reviewers read top-to-bottom and skip PRs that hide their context.
+-->
+
+## Summary
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)